com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -30,45 +30,47 @@ public static class CustomMessagingManager
         /// </summary>
         public static event UnnamedMessageDelegate OnUnnamedMessage;
 
-        internal static void InvokeUnnamedMessage(ulong clientId, Stream stream)
+        internal static void InvokeUnnamedMessage(NetworkingManager networkingManager, ulong clientId, Stream stream)
```

Comment:
When I look at this function I really wonder what if we mad this an instance method off of NetworkingManager?  It's weird to have a static method that takes an instance and then does most of its work with that instance.